### PR TITLE
[FIX] mail: remove unused orm service

### PR DESCRIPTION
**Pull request by phenix-factory**

### CODE REVIEW

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by seb-odoo** [comment]
It looks like it is actually used?

File: addons/mail/static/src/core/web/thread_actions.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ threadActionsRegistry
             return component.thread.isEmpty;
         },
         open(component) {
-            component.orm.silent.call("mail.message", "mark_all_as_read");
+            component.thread.markAllMessagesAsRead();
```
Comment (seb-odoo) [inline addons/mail/static/src/core/web/thread_actions.js:16]: This is not the same, `mark_all_as_read` (no param) is inbox messages.
thread is specific (for real thread, it doesn't work for inbox).

Comment (seb-odoo) [reply addons/mail/static/src/core/web/thread_actions.js:16]: The `orm` service is simply still used, this PR doesn't really make sense.

Comment (alexkuhn) [reply addons/mail/static/src/core/web/thread_actions.js:16]: The PR can define `useService("orm")` in this thread action, so the locality of concern is better.
This is not really worth a FIX in stable IMO, but whatever ;)

**Pull request closed without merging**